ray-curator/ray_curator/stages/download/text/wikipedia/README.md

```diff
@@ -0,0 +1,131 @@
+# Wikipedia Download and Extract Stage
```

Comment:
Can nuke this imo.. let that be a separate documentation thing? wdyt @sarahyurick